CHANGELOG.develop

```diff
@@ -338,6 +343,17 @@ Benner and Paweł Siudak):
   Jensen)
 - Added optional =append= and =local= parameters to =spacemacs/add-to-hooks=
   (thanks to Sylvain Benner)
+- Support winner, ysnappet for portable dumper (thanks to Sylvain
```

Comment:
Typo: "ysnappet" should be "yasnippet".  Actually though, I have been skipping entries for dump support for individual packages since I consider such changes to be fixes to the dump support itself.